src/Infrastructure/Persistence/Doctrine/Migrations/Version20240115141540.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240115141540 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE location ADD road_type VARCHAR(40) DEFAULT NULL');
+        $this->addSql('ALTER TABLE location ADD administrator VARCHAR(255) DEFAULT NULL');
+        $this->addSql('ALTER TABLE location ADD road_number VARCHAR(50) DEFAULT NULL');
+        $this->addSql('ALTER TABLE location ALTER city_code DROP NOT NULL');
+        $this->addSql('ALTER TABLE location ALTER city_label DROP NOT NULL');
+
+        $this->addSql("UPDATE location SET road_type = 'lane'");
+        $this->addSql('ALTER TABLE location ALTER road_type SET NOT NULL');
+
+        $this->addSql('ALTER TABLE location_new ALTER city_code DROP NOT NULL');
+        $this->addSql('ALTER TABLE location_new ALTER city_label DROP NOT NULL');
+        $this->addSql('ALTER TABLE location_new ADD road_type VARCHAR(40) DEFAULT NULL');
+        $this->addSql('ALTER TABLE location_new ADD administrator VARCHAR(255) DEFAULT NULL');
+        $this->addSql('ALTER TABLE location_new ADD road_number VARCHAR(50) DEFAULT NULL');
```

Comment:
(Détail)

Ça peut aussi paraître être un pinaillage, mais comme location et location_new sont sensés être de véritables "miroirs", en relisant comme l'ordre n'est pas le même on doit prêter davantage attention visuelle pour s'apercevoir s'il y a une erreur ou si on a bien ajouté les mêmes champs

```suggestion
        $this->addSql('ALTER TABLE location_new ADD road_type VARCHAR(40) DEFAULT NULL');
        $this->addSql('ALTER TABLE location_new ADD administrator VARCHAR(255) DEFAULT NULL');
        $this->addSql('ALTER TABLE location_new ADD road_number VARCHAR(50) DEFAULT NULL');
        $this->addSql('ALTER TABLE location_new ALTER city_code DROP NOT NULL');
        $this->addSql('ALTER TABLE location_new ALTER city_label DROP NOT NULL');
```